pkg/util/utils.go

```diff
@@ -421,24 +421,25 @@ func connectStrings(sep string, parts ...string) string {
 	return strings.Join(strParts, sep)
 }
 
-func GenerateDisplayName(res_name, prefix, suffix, project, cluster string) string {
-	// Return a string in this format: prefix-cluster-res_name-project-suffix.
-	return connectStrings("-", prefix, cluster, res_name, project, suffix)
+func GenerateDisplayName(connector, resName, prefix, suffix, project, cluster string) string {
```

Comment:
Not introduced by you, but it looks we can make this function as private since it is only called in `GenerateTruncName`?

Reply:
Good suggestion, Thanks. I made this function as a private function since not called by external world.